src/components/AutoComplete/AsyncAutoComplete.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { AsyncTypeahead } from 'react-bootstrap-typeahead';
+import PropTypes from 'prop-types';
+
+class AsyncAutoComplete extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
optional: can be written as
```javascript
class AsyncAutoComplete extends React.Component {
    state = {
        options: []
    };
}
```
constructor can be removed as well if you move to this.

Reply:
@sharvit again here - @dmiller9911 has a good point. The `state = {}` syntax is shorter than setting all state in the `constructor()` and I know many React communities have moved this direction. Do you have a strong preference either way there? This would also be a new convention we could migrate towards and open a new issue for...
 
@dmiller9911 really appreciate the fresh 👀's on all of this!! ... 💯